src/app.py

```diff
@@ -1,13 +1,33 @@
 from dash import Dash, html, dcc, Input, Output
 import pandas as pd
 import dash_bootstrap_components as dbc
+import ast
 
 # Initiatlize the app
 app = Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
 server = app.server
 tracks_df = pd.read_csv('data/raw/tracks_processed.csv')
 
+
+# convert stringified lists into actual lists
+def convert_string_to_list(string):
+    try:
+        return ast.literal_eval(string)
+    # Return an empty list if the string is not a valid list representation
+    except ValueError:
+        return []  
+        
+tracks_df['genres'] = tracks_df['genres'].apply(convert_string_to_list)
+unique_genres = sorted(set(genre for sublist in tracks_df['genres'] for genre in sublist))
+genre_dropdown_options = [{'label': genre, 'value': genre} for genre in unique_genres]
+
 # Configuration
+genre_dropdown = dcc.Dropdown(
```

Comment:
Looks great! Before merging this branch are you adding the `genre_dropdown` to our `app.layout`?